cypress/cypress-smoketests/support/email-receipt.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  DeleteObjectsCommand,
+  GetObjectCommand,
+  ListObjectsV2Command,
+  S3Client,
+} from '@aws-sdk/client-s3';
```

Comment:
Could you move this file to our shared cypress functions? Maybe make an s3 folder?

cypress/helpers/s3

Reply:
Tempted to, but I don't believe this _should_ be shared. It's smoketest specific, and shouldn't be used in readonly or integration tests

Reply:
Could probably be convinced otherwise